CHANGELOG.md

```diff
@@ -2,7 +2,83 @@
 
 ## Unreleased
 
-  - …
+  - General data model
+    - `DataPayload` no longer needs a lifetime (#1297, #1279)
+    - Re-write ResourceKey (#1511)
+    - Rewrite ErasedDataProvider as AnyProvider (#1495)
+    - Add EitherProvider and rename IterableDataProviderCore to IterableProvider (#1455)
```

Comment:
Thought: I think the changelog should be more high-level: things users care about.  I think this is too detailed.  (I will not leave this comment on future bullet points; it would be easier to make the changes in bulk, if you agree)

Reply:
So to be clear I've left out a lot of  PRs, this is mostly stuff with user visible impact. I know that users don't use IterableProvider so I'm fine cleaning up this bullet point to not mention it.


99% of the time I look at a changelog it is when I'm performing an upgrade, and either I want to prepare for changes or I want to understand a breakage, and this level of detail is often necessary. I'm happy to have a higher level summary that's only a couple bullet points up top (and i'm happy to do things like scrubbing mentions of IterableProvider, etc), but I do think this is the right level of detail.